backend/corpora/lambdas/handle_success/requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg2-binary>=2.8.5
```

Comment:
These versions have been superseded, but thought they should be kept in sync with `backend/corpora/upload_failures/requirements.txt`.

The package versions should all be updated at the same time.